specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/virtualNetworkGateway.json

```diff
@@ -362,6 +362,52 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkGateways/{virtualNetworkGatewayName}/resetvpnclientsharedkey": {
+      "post": {
+        "tags": [
+          "VirtualNetworkGateways"
+        ],
+        "operationId": "VirtualNetworkGateways_Resetvpnclientsharedkey",
```

Comment:
Please add an example for this operation like this one: https://github.com/Azure/azure-rest-api-specs/pull/3262/files#diff-b23c5faf0d788f07c5a5e46e20fa6cf6R457. The schema of examples can be found here: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md#structure-of-the-extension.